src/main/java/de/galan/commons/time/Instants.java

```diff
@@ -30,11 +30,15 @@
 public class Instants {
 
 	/*
-	TemporalAdjuster
+	TODO Check TemporalAdjuster
 	 */
 
-	public static final String DATE_FORMAT_LOCAL = "yyyy-MM-dd HH:mm:ss[.SSS]";
-	public static final String DATE_FORMAT_UTC = "yyyy-MM-dd'T'HH:mm:ss[.SSS]'Z'";
+	public static final String DATE_FORMAT_LOCAL_INPUT = "yyyy-MM-dd HH:mm:ss[.SSSSSS][.SSS]";
```

Comment:
some date formatters (default golang `time.Date` for example) use an even higher accuracy.
The formatted `time.Now() -> "2019-09-30 14:26:42.586787248 +0200 CEST"`. But maybe we can ignore this here.